pkg/assets/bindata.go

```diff
@@ -69354,17 +69371,14 @@ var _views_builds_html = []byte(`<div ng-controller="ProjectController" class="c
 </div>
 <div ng-switch-when="Docker">
 <div ng-if="buildConfig.spec.strategy.dockerStrategy.from">
-<dl class="dl-horizontal">
```

Comment:
The wrapping <dl> shouldn't be here since the <dt> & <dd> are already within he higher level <dl class="dl-horizontal left indent">  and was causing extra bottom margin.